package.json

```diff
@@ -48,6 +48,7 @@
     "cozy-stack-client": "^27.21.0",
     "cozy-ui": "^70.4.2",
     "date-fns": "2.28.0",
+    "es-abstract": "1.20.1",
```

Comment:
Usually, when we need to add `es-abstract` within the package.json, this is because we have something that goes wrong somewhere with our dependencies graph. It doesn't smell good

Reply:
Yes, that's the issue, but `yarn why` prints so many `es-abstract` I was a bit overwhelmed. Should we just alias it in the Webpack config?

Reply:
We have no choice for now because node-polyglot for instance requires absolute pathed es-abstract instead of relative pathed. So it tries to find something that doesn't exist, unless we add es-abstract to the project.